### PR TITLE
MGMT-13337: Adding monthly report

### PR DESCRIPTION
**Pull request by danmanor**
Env config at: https://gitlab.cee.redhat.com/service/app-interface/-/merge_requests/56338

### CODE REVIEW

**Review by osherdp — APPROVED** [review]
looks ok to me

File: src/jobsautoreport/main.py [hunk 1]
```diff
@@ -20,19 +22,34 @@ def main() -> None:
...
+            year=now.year,
+            month=now.month,
+            day=now.day,
+            hour=0,
+            minute=0,
+            second=0,
+            microsecond=0,
+            tzinfo=timezone.utc,
+        )
+        report_start_time = report_end_time - relativedelta(months=1)
```
Comment (adriengentil) [inline src/jobsautoreport/main.py:52]: that library looks cool :+1:

**Review by adriengentil — APPROVED** [review]

**Pull request merged**